src/reactapp/src/components/items/components/CartItemsFormManager/utility/index.js

```diff
@@ -1,14 +1,43 @@
+import { number as YupNumber, object as YupObject } from 'yup';
+
+import { __ } from '../../../../../i18n';
+import { _keys } from '../../../../../utils';
+
 export function prepareCartItemsUniqueId(cartItems) {
   return cartItems.map(({ id, quantity }) => `${id}_${quantity}`).join('__');
 }
 
-export function prepareCartItemsFormData(cartItems) {
+export function prepareCartItemFormikData(cartItems) {
   return cartItems.reduce((accumulator, item) => {
     const cartItemId = parseInt(item.id, 10);
-    accumulator[`item__${cartItemId}`] = {
-      cart_item_id: cartItemId,
-      quantity: parseFloat(item.quantity),
-    };
+    accumulator[`${cartItemId}_qty`] = parseFloat(item.quantity);
     return accumulator;
   }, {});
 }
+
+export function prepareCartItemsValidationSchema(cartItemFormData) {
+  const validationSchema = {};
+
+  _keys(cartItemFormData).forEach((itemKey) => {
+    validationSchema[itemKey] = YupNumber()
+      .required(__('Required'))
+      .min(1, __('Qty should be > 0.'));
```

Comment:
"regular humans" wouldn't always understand the greater than symbol. "Qty should be a number higher than 0" maybe?

Reply:
@wigman This will break the styles in the cart section. because as you know cart items have kind of table structure. There is no space to show a longer message there unfortunately. I could not think of a concise, compact message to show there

Reply:
Does "Should be a positive number" fit? Otherwise, "Positive number required"?
Worst case: use "Qty is invalid"

This is going to be an issue with translations anyhow, other languages will require longer text there. Let's look at that later, but for now see if the above fits?

Reply:
Used "Invalid qty"